feat/feature-flags.json

```diff
@@ -11,10 +11,10 @@
     "new_header": {
       "status": {
         "staging": "enabled",
-        "production": "disabled"
+        "production": "enabled"
       },
       "description": "Toggle the new header, including recent searches.",
-      "defaultState": "off"
+      "defaultState": "on"
```

Comment:
```suggestion
```
I think you only need `defaultState` when the flag is switchable.

Reply:
Yes, `defaultState` has no effect when status is 'enabled'.